src/hotspot/os/bsd/os_bsd.cpp

```diff
@@ -121,6 +121,10 @@
 #include <lwp.h>
 #endif
 
+#ifdef __OpenBSD__
```

Comment:
It looks like this was transferred from os_bsd_x86.cpp where the conditional was

```
#if !defined(__APPLE__) && !defined(__NetBSD__)
```

Is there a reason for not keeping it the same?

Reply:
Hi Greg,

Thanks for the review. I take it from that conditional that just FreeBSD and OpenBSD need pthread_np.h. There was a conditional already in os_bsd.cpp that covers FreeBSD a bit earlier in the file (that also appears to cover pre 9.x):
```
#ifdef __FreeBSD__
# if __FreeBSD_version > 900030
#  include <pthread_np.h>
# else
#  include <sys/thr.h>
# endif
# include <sys/cpuset.h>
# include <vm/vm_param.h>
#endif
```
So I added a new ifdef OpenBSD to add the missing pthread_np.h for it. I probably should have covered that in the commit message.

Reply:
Thanks for clarifying Kurt.  That is reasonable.  I should likely collapse that earlier code a little since it seems unlikely that overall the code base supports FreeBSD <= 9 (or that we would want it to).